addons/website_autocomplete/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_autocompletion_provider_web_admin,website_sale_autocompletion_provider_publisher,model_autocompletion_provider,base.group_system,1,1,1,1
```

Comment:
Hmm I wonder if we really want `perm_create` and `perm_unlink`, since this model should not be managed on it's own.
But I guess it's ok as is.